test/support/test_network_adapter.ex

```diff
@@ -116,8 +116,8 @@ defmodule GrizzlyTest.InclusionAdapter do
   def grant_s2_keys(_s2_keys, state) do
     new_state = cancel_timer(state)
 
-    {:ok, command} =
-      NodeAddDSKReport.new(seq_number: 1, input_dsk_length: 5, dsk: DSK.parse_pin("12345"))
+    {:ok, dsk} = DSK.parse_pin("12345")
```

Comment:
How did this ever work?

Reply:
Seems like it would only raise an error if we tried to binary encode the NodeAddDSKReport, but Grizzly should only ever receive them.

In the inclusion test cases, we weren't making an assertion on this value. It wasn't really a problem because in practice, we would only ever decode this command, and this is test-only code anyway.

In any case, I added an assertion that would fail on main.